omod/src/main/java/org/openmrs/module/owa/web/controller/OwaRestController.java

```diff
@@ -47,164 +47,175 @@
 @Deprecated
 @Controller
 public class OwaRestController {
-	
+
 	private static final Log log = LogFactory.getLog(OwaRestController.class);
-	
+
 	// -------------------------------------------------------------------------
 	// Dependencies
 	// -------------------------------------------------------------------------
 	@Autowired
 	private AppManager appManager;
-	
+
 	@Autowired
 	private MessageSourceService messageSourceService;
-	
+
 	// -------------------------------------------------------------------------
 	// REST implementation
 	// -------------------------------------------------------------------------
 	@RequestMapping(value = "/rest/owa/applist", method = RequestMethod.GET)
-		@ResponseBody
-		public List<App> getAppList() {
-			List<App> appList = new ArrayList<>();
-	        if(Context.hasPrivilege("Manage OWA")){
-	        	appManager.reloadApps();
-	        	appList = appManager.getApps();
-	        }
-	        return appList;
+	@ResponseBody
+	public List<App> getAppList() {
+		List<App> appList = new ArrayList<>();
+		if(Context.hasPrivilege("Manage OWA")){
+			appManager.reloadApps();
+			appList = appManager.getApps();
 		}
-	
+		return appList;
+	}
+
 	@RequestMapping(value = "/rest/owa/settings", method = RequestMethod.GET)
-        @ResponseBody
-        public List<GlobalProperty> getSettings() {
-            List<GlobalProperty> owaSettings = new ArrayList<>();
-            if(Context.hasPrivilege("Manage OWA")){
-                    owaSettings.add(Context.getAdministrationService().getGlobalPropertyObject(AppManager.KEY_APP_FOLDER_PATH));
-                    owaSettings.add(Context.getAdministrationService().getGlobalPropertyObject(AppManager.KEY_APP_BASE_URL));
-                    owaSettings.add(Context.getAdministrationService().getGlobalPropertyObject(AppManager.KEY_APP_STORE_URL));
-            }
-            return owaSettings;
-        }
-	
+	@ResponseBody
+	public List<GlobalProperty> getSettings() {
+		List<GlobalProperty> owaSettings = new ArrayList<>();
+		if(Context.hasPrivilege("Manage OWA")){
+			owaSettings.add(Context.getAdministrationService().getGlobalPropertyObject(AppManager.KEY_APP_FOLDER_PATH));
+			owaSettings.add(Context.getAdministrationService().getGlobalPropertyObject(AppManager.KEY_APP_BASE_URL));
+			owaSettings.add(Context.getAdministrationService().getGlobalPropertyObject(AppManager.KEY_APP_STORE_URL));
+		}
+		return owaSettings;
+	}
+
 	@RequestMapping(value = "/rest/owa/settings", method = RequestMethod.POST)
-        @ResponseBody
-        public List<GlobalProperty> updateSettings(List<GlobalProperty> settings) {
-            List<GlobalProperty> owaSettings = new ArrayList<>();
-            if(Context.hasPrivilege("Manage OWA")){
-                if (null != settings) {
-                    for (GlobalProperty gp : settings) {
```

Comment:
Did you actually remove all these?

Reply:
Due to @Charpell's comment, I removed the indentation issues that were in the files. Those were brought in by the previous code so when I resolved the issues, it shows that the whole block of code was removed.